tools/update_boots.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Script to automatically update all or a select number of boot tools
+
+if [ -z "$boottools" ]; then
+  echo "Updating all boot tools"
+  boottools="meta,curl,make,git,less,perl,jq,bash,diffutils,findutils,coreutils,tar,gzip,xz,bzip2,vim,ncurses"
```

Comment:
with the changes to use the tar -axf service, do we need gzip, xz, bzip2 in boot? Probably good for this rev but maybe we can remove it next time?

Reply:
Ah